Makefile

```diff
@@ -41,13 +41,9 @@ gen: api_docs swagger
 
 deps:
 	@GO111MODULE=off go get -u github.com/myitcv/gobin
-	@gobin github.com/go-swagger/go-swagger/cmd/swagger@v0.23.0
-	@gobin github.com/codeskyblue/fswatch
+	@gobin github.com/go-swagger/go-swagger/cmd/swagger@v0.24.0
 	@gobin github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
 
-watch:
```

Comment:
remove the fswatch as the project was not updated for a while and it reads too many file descriptors.